ols/constants.py

```diff
@@ -176,8 +176,7 @@ class QueryValidationMethod(StrEnum):
 GPT35_TURBO = "gpt-3.5-turbo"
 
 # embeddings metadata
-EMBEDDINGS_ROOT_DIR = "/workdir/ocp-product-docs-plaintext"
-OCP_DOCS_ROOT_URL = "https://docs.openshift.com/container-platform"
+OCP_DOCS_ROOT_URL = "https://docs.openshift.com/container-platform/"
```

Comment:
since this is only used in test code, please move it there.

(or at least it should only be being used in test code.  if it's being used in OLS code after https://github.com/openshift/lightspeed-rag-content/pull/3 then we've missed something)